cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -28,13 +28,13 @@ import (
 	apiserveroptions "k8s.io/apiserver/pkg/server/options"
 	"k8s.io/apiserver/pkg/storage/storagebackend"
 	utilconfig "k8s.io/apiserver/pkg/util/flag"
+	restclient "k8s.io/client-go/rest"
 	kapi "k8s.io/kubernetes/pkg/api"
 	kubeoptions "k8s.io/kubernetes/pkg/kubeapiserver/options"
 	kubeletclient "k8s.io/kubernetes/pkg/kubelet/client"
 )
 
 func TestAddFlagsFlag(t *testing.T) {
```

Comment:
another nit: it's strange the test name was `TestAddFlagsFlag`. I think `TestAddFlags` would be ok. WDYT?